tests/Feature/WebhooksTest.php

```diff
@@ -21,7 +21,7 @@ public function test_gracefully_handle_webhook_without_alert_name()
 
     public function test_it_can_handle_a_payment_succeeded_event()
     {
-        Event::fake();
+        Cashier::fake();
```

Comment:
Changing these here isn't needed. Only the tests where a Http::fake was used should be replaced. The rest should still hit Paddle's API.

Reply:
So the way I have it set up is that `Cashier::fake()` fakes the events fired by Cashier (with the exception of the webhook events) as well as the Http calls. Confirming you would like me to remove the faked events from this and pull out all the assertions I added in as well.

Reply:
I see what you did now and that's actually pretty neat. Let's keep that 👍